### ISSUE TITLE
Choose best class for digestion concept

### ISSUE THREAD

**Issue by FrancescoVit** [FoodOntology/joint-food-ontology-wg#5]
*This a sort of prototype for issues raising from group calls. Maybe in the main page we could set guidelines for contribution template? Let's also discuss this @ddooley* 

**ISSUE:** choose best term for the unified ontology to describe the digestion concept. Two alternatives were proposed, one from ONS and another from GO. Issue arising during virtual meeting on 27/05/2020.

**Option A**
ONTOLOGY : ONS, curator (refer if in work group) @FrancescoVit 
ID: ONS:0000101
LABEL: Food digestion
DEFINITION: Digestion is the breakdown of large insoluble food molecules into small water-soluble food molecules. The breakdown is both mechanical and chemical (enzymatic digestion). During digestion, food components are released from the food matrix and become bioaccessible. Bioaccessible molecules can be absorbed.

**Option B**
ONTOLOGY: GO
ID: GO:0007586
LABEL: digestion
DEFINITION: The whole of the physical, chemical, and biochemical processes carried out by multicellular organisms to break down ingested nutrients into components that may be easily absorbed and directed into metabolism.

**Comment by ddooley** [comment]
Looks like a good start for the terminology question.  There may be room for both terms if one is subordinate to the other.  I recall someone raising in the meeting the scope of what digestion encompasses.  One sense is pitched specifically to food nutrients - I believe that is the ONS target - while the other, encompasses a wider variety of inputs and outputs?  Does the intake and absorption of water count in digestion process or not?  If not, what is that then classified as? Are there other compounds that aren't quite considered "nutrients" or that aren't broken down, that are absorbed and directed into metabolism / are bioaccessible?  Is caffeine considered a nutrient that is digested?

**Comment by maweber-bia** [comment]
I prefer the general definition of GO for digestion for the following reasons: 1)Basically, digestion is the mechanical and chemical transformation of food into nutrients that **may or may not** be assimilated by the body. 2) By definition, digestion is a process present in all heterotrophic organisms and takes place in a digestive system. However, I would like to add that we also have some experimental "in vitro digestion" studies that should be taken into account.  
We should also distinguish  bioavailability versus bioaccessibility and bioactivity with regard to the absorption and metabolism. Hope this could help !

**Comment by maweber-bia** [comment]
In addition,  Wikipedia's definition of a nutrient is the following  : A nutrient is a substance used by an organism to survive, grow, and reproduce. 
We should  distinguish between macro- vs micro-nutrients, essential vs non-essential or conditionnally essential. Compounds such as phytochemicals or caffeine could be taken as non-nutrients.

**Comment by FrancescoVit** [comment]
> 1)Basically, digestion is the mechanical and chemical transformation of food into nutrients that **may or may not** be assimilated by the body.

Agree on that

> In addition, Wikipedia's definition of a nutrient is the following : A nutrient is a substance used by an organism to survive, grow, and reproduce. We should distinguish between macro- vs micro-nutrients, essential vs non-essential or conditionnally essential. Compounds such as phytochemicals or caffeine could be taken as non-nutrients.

In ONS, we "solved" this issue by introducing the "Food component entity" class (ONS:0000073) intending "something in the food, not necessarily a nutrient". The class has 4 sub-classes ("additive", "contaminant", "food bioactive", and "nutrient") and connected to the role "food component" in CHEBI (CHEBI:78295). We could further detail sub-classes of macro- and micro-nutrient class



>  There may be room for both terms if one is subordinate to the other. 

To me, the GO term seems more general, but then define "break down ingested nutrients into components that may be easily absorbed and directed into metabolism." which is narrowing a bit the spectrum to only nutrients

Equivalence between classes could do the trick?

pinging it also to @Agnes-5

**Comment by LCCarmody** [comment]
My inclination is to go with GO's digestion term, but I think that there is an argument to make that the 'food digestion' term is a child of digestion for those who need a more specific term.

**Comment by maweber-bia** [comment]
> 
> > In addition, Wikipedia's definition of a nutrient is the following : A nutrient is a substance used by an organism to survive, grow, and reproduce. We should distinguish between macro- vs micro-nutrients, essential vs non-essential or conditionnally essential. Compounds such as phytochemicals or caffeine could be taken as non-nutrients.
> 
> In ONS, we "solved" this issue by introducing the "Food component entity" class (ONS:0000073) intending "something in the food, not necessarily a nutrient". The class has 4 sub-classes ("additive", "contaminant", "food bioactive", and "nutrient") and connected to the role "food component" in CHEBI (CHEBI:78295). We could further detail sub-classes of macro- and micro-nutrient class
> 
> > There may be room for both terms if one is subordinate to the other.
> 
I am not an expert so far in ontology design (I am learning based on practice ) but it would definitely be great if  the 4 categories you mentionned are taken on board and connected to their respective "role".  That is a big point.


> To me, the GO term seems more general, but then define "break down ingested nutrients into components that may be easily absorbed and directed into metabolism." which is narrowing a bit the spectrum to only nutrients
> 
Yes this is narrowing the spectrum to nutrients, but I do not see the ONS definition for food digestion telling otherwise : During digestion, food components are released from the food matrix and become bioaccessible. Bioaccessible molecules can be absorbed.

But it is perhaps my understanding of the nuance that is weak...

**Comment by ddooley** [comment]
If digestion encompasses "non-nutrients" then the GO definition seems a bit limiting in mentioning only nutrients.
The ONS definition mentions  "large insoluble food molecules" as the input, and, along with Wikipedia, a "small water soluble food molecules" as the output. Yet the food molecule part also carries with it the "nutrient" dependancy?
I'll ask the FooDb folks, that are examining many other kinds of ingestible molecules present in foods (e.g. [cafine](https://foodb.ca/compounds/FDB002100)), what their needs are visa vis a definition. They have over 70,000 compounds listed, which makes me leery of limiting the digestion definition to "large insoluble food molecules". 
@pcastellanoescuder your lab must consider the digestion of many other molecules besides the classical food nutrients, right?

**Comment by pcastellanoescuder** [comment]
Yes, we study both compounds derived directly from the diet (present in food) and compounds derived from the transformation of the first ones. FOBI is made up of a mixture of both. However, I'm not sure if in this case the appropriate term should be "digestion" or "transformation".

Referring to the term to define "digestion" I think that as @ddooley proposes, it might be a good approach to use the GO term as a higher and more generic term than the ONS term. I think they are not exclusive since the term GO can be much more generic.

On the other hand, should we consider adding a term for the concept "transformation"? It could be a term associated with "digestion" but conceptually different... Could it be useful?

**Comment by maweber-bia** [comment]
> On the other hand, should we consider adding a term for the concept "transformation"? It could be a term associated with "digestion" but conceptually different... Could it be useful?

Yes, it would be useful to add the concept of transformation as a process that could lead to the formation of new molecules , i.e "neoformed" molecules, but I think this should not be included in the definition of digestion which is basically the process of **molecular simplification**, i.e. the transformation of macromolecules (large molecules) into micromolecules. 


> If digestion encompasses "non-nutrients" then the GO definition seems a bit limiting in mentioning only nutrients.
> The ONS definition mentions "large insoluble food molecules" as the input, and, along with Wikipedia, a "small water soluble food molecules" as the output. Yet the food molecule part also carries with it the "nutrient" dependancy?

Once again, I think that it is incorrect to extend the scope of digestion outside its primary physiological definition : Digestion is the mechanical and chemical transformation of food into nutrients that **may or may not** be assimilated by the body (here "body" could be replaced by "**heterotrophic organism**" for a more general definition see issue #9  )

> I'll ask the FooDb folks, that are examining many other kinds of ingestible molecules present in foods (e.g. [cafine](https://foodb.ca/compounds/FDB002100)), what their needs are visa vis a definition. They have over 70,000 compounds listed, which makes me leery of limiting the digestion definition to "large insoluble food molecules".
> @pcastellanoescuder your lab must consider the digestion of many other molecules besides the classical food nutrients, right?

In my opinion, it would be useful to consider the concept of "**ingestible molecule**" to include all type of molecules, ie food components including nutrients and non-nutrients, neoformed, additives, contaminants, etc.

**Comment by ddooley** [comment]
Ok, I understand now, by definition the absorption of other molecules such as caffeine is not covered as part of the digestion process.

**Comment by maweber-bia** [comment]
right! In this case, one should consider the concept "absorption" instead of "digestion"

**Comment by FrancescoVit** [comment]
> right! In this case, one should consider the concept "absorption" instead of "digestion"

The "adsorption" of something is included both in ONS and in GO definition, but maybe it should be removed if the idea is that absorption is inherent in digestion, but that there are some molecules which are adsorbed but not digested.

To sum up, we are talking about the fate of what was ingested during **eating** process, and that is likely brought by **food**, does it sound right? I like the **ingestible molecule** concept, but it sound a bit weird to connect a role (i.e. nutrient) to some molecules, only by the fact that they are ingestible. Shouldn't we consider them nutrient also if not ingested? 

In my opinion, we could think of whichever molecule in food as **food component**, which contains different sub-classes connected to different roles (i.e. **nutrient** subclass has the **nutrient role**, nutrient subclass has Carbohydrates as sub-class, connected to CHEBI). To return to digestion, we could use the GO definition, and define that **ingested nutrients** (one of the output of the **eating process**) represent the input of **digestion process**. On the other hand, there are some **ingested food components** which does not enter the **digestion process** but rather enter an **absorption process**

Hope to have this stream of thought right in written form...

**Comment by ddooley** [comment]
So I am really appreciating the differentiation happening here as it directly affects/clarifies the top level nutrient hierarchy and gets at both nutrients and "non-nutrients" which are both indispensable in health analysis. We can finish off the digestion process and absorption process definitions here.  But I'm going to start an issue to define the matching molecular food component classes which a nutrition hierarchy is pretty much at the top of.

**Comment by cmrn-rhi** [comment]
> The "adsorption" of something is included both in ONS and in GO definition, but maybe it should be removed if the idea is that absorption is inherent in digestion, but that there are some molecules which are adsorbed but not digested.

Just to clarify, when referencing "adsorption" are you meaning "absorption"?

> To sum up, we are talking about the fate of what was ingested during eating process, and that is likely brought by food, does it sound right? I like the ingestible molecule concept, but it sound a bit weird to connect a role (i.e. nutrient) to some molecules, only by the fact that they are ingestible. Shouldn't we consider them nutrient also if not ingested?

True, when using the CHEBI definition of **[nutrient](http://purl.obolibrary.org/obo/CHEBI_33284)** (A nutrient is a food component that an organism uses to survive and grow.) and/or OBI's **[nutrient role](http://purl.obolibrary.org/obo/OBI_0000204)** (a role that inheres in a material entity and is realized in the use of that material entity by an organism when it is used in that organism's metabolism and provides nourishment.) a nutrient is not strictly ingestible. 

Examples: **Vitamin D** via sun exposure or **Transdermal Magnesium**

**Comment by maweber-bia** [comment]
As already pointed out by Rhiannon, absorption should not be confused with adsorption.

I like the definition of a nutrient being a food component that an organism uses to survive and grow, and it is true that nutrients can also be absorbed by parenteral route or blood injection, and not by ingestion (oral route)  only.

I totally agree with the proposal of Francesco :
"In my opinion, we could think of whichever molecule in food as food component, which contains different sub-classes connected to different roles (i.e. nutrient subclass has the nutrient role, nutrient subclass has Carbohydrates as sub-class, connected to CHEBI). To return to digestion, we could use the GO definition, and define that **ingested nutrients (one of the output of the eating process) represent the input of digestion process.** On the other hand, there are some ingested food components which does not enter the digestion process but rather enter an absorption process"

**Comment by ddooley** [comment]
I'm adding a "food component absorption process" term which sits alongside "digestion" in the [Dietary Terms](https://docs.google.com/spreadsheets/d/18Ml5pLHfd0IVGD3roxF0NkjXRry0rRrsa8Y06y2fgr0/edit#gid=0) sheet - it could use a definition.  Meanwhile the molecule / nutrient list is growing in the  [Molecular Food Component Hierarchy](https://docs.google.com/spreadsheets/d/1kNcTyGLYqpmh9_b03aYe221YkWKD0hsve6uIftGQEzc/edit#gid=0) sheet. There are a number of duplicate terms probably in this sheet between foodon and other ontologies so its to be expected that we'll have to winnow them down.
